index.bs

```diff
@@ -1042,6 +1046,10 @@ The <dfn attribute for="XRRenderState">depthNear</dfn> attribute defines the dis
 
 Note: Typically when constructing a perspective projection matrix for rendering the developer specifies the viewing frustum and the near and far clip planes. When displaying to an [=immersive XR device=] the correct viewing frustum is determined by some combination of the optics, displays, and cameras being used. The near and far clip planes, however, may be modified by the application since the appropriate values depend on the type of content being rendered.
 
+The <dfn attribute for="XRRenderState">pasthroughFullyObscured</dfn> attribute is a hint to the {{XRSystem}} from the author to indicate that the pixels of the immersive experience completely cover the viewport.
+
+NOTE: the {{XRSystem}} MAY use this to temporarily disable passthrough to save system resources.
```

Comment:
I think we should add more non-normative text here to explain that this flag does not guarantee that the user's environment is not visible, especially in the case of optically transparent displays.

Reply:
I think the "MAY" already indicates this but I added some more prose.